ui/src/timeMachine/components/Vis.tsx

```diff
@@ -103,6 +106,7 @@ const TimeMachineVis: SFC<Props> = ({
               files={files}
               loading={loading}
               properties={resolvedViewProperties}
+              check={check}
```

Comment:
I wonder if we would be better off connecting the `CheckPlot` to redux, rather than drilling the check from

```
Vis -> ViewSwitcher -> CheckPlot
```

and 
```
Cell -> View -> RefreshingView -> ViewSwitcher -> CheckPlot
```

I'm worried that all the intermediate stages will provide appealing places to introduce check-related logic in the future, even if that logic would be better off in redux or elsewhere.